example/spec/ingestGranule/IngestGranule.UMM.output.payload.template.json

```diff
@@ -7,6 +7,8 @@
       "version": "006",
       "published": true,
       "cmrLink": "https://cmr.uat.earthdata.nasa.gov/search/granules.json?concept_id=G1222231611-CUMULUS",
+      "cmrConceptId": "G1222231611-CUMULUS",
+      "cmrFileType": "umm_json_v1_5",
```

Comment:
Can we change this `cmrFileType` to something more descriptive, or less deceptive?
is `cmrMetadataType` more accurate? `cmrMetadataFormat` `cmrFileFormat`

Reply:
`metadataNativeFormat`?  I'm not good at naming.

Reply:
I think I realize the irony that isUMMGFileType is my function.

Reply:
I'm fine with `cmrMetadataType`. I'll change it

Reply:
Also, on a closer look, if I changed `cmrFileType` to `cmrMetadataFormat` everywhere, I'd have to update the config for the sips workflow as well:

https://github.com/nasa/cumulus/blob/be59452b0134a8298d51796bada2d013fa92dfda/example/workflows/sips.yml#L192

Does that seem like a problem?

Reply:
Yes, you'd have to update it there.  But no one should be using it yet since it's new.